include/mqtt_client.h

```diff
@@ -191,6 +191,7 @@ typedef struct {
     int refresh_connection_after_ms;        /*!< Refresh connection after this value (in milliseconds) */
     const struct psk_key_hint *psk_hint_key;     /*!< Pointer to PSK struct defined in esp_tls.h to enable PSK authentication (as alternative to certificate verification). If not NULL and server/client certificates are NULL, PSK is enabled */
     bool          use_global_ca_store;      /*!< Use a global ca_store for all the connections in which this bool is set. */
+    esp_err_t ((*crt_bundle_attach)(void *conf)); /*!< Use ESP x509 Certificate Bundle for mqtts
```

Comment:
the closing `*/` part of the comment seems to be missing

Reply:
thank you for your feedback. I fixed it in the new commit.